daemon/api_download.go

```diff
@@ -85,7 +86,8 @@ func streamOneSnap(c *Command, user *auth.UserState, snapName string) Response {
 
 	return fileStream{
 		SnapName: snapName,
-		Info:     info,
+		FileName: filepath.Base(info.MountFile()),
```

Comment:
nitpick: we are not super consistent, but Filename is by far the more common spelling in the codebase